conbench/buildinfo.py

```diff
@@ -14,13 +14,17 @@ class Buildinfo:
     commit: str
     # Name of the branch carrying the commit.
     branch_name: str
-    # Example value: 2023-02-06 12:01:01+00:00
+    # Example value: 2023-02-06 12:01:01Z
```

Comment:
Both are valid in RFC3339.

For the record: in the last few years I got to appreciate that RFC3339 has overall less flexibility than ISO 8601, but that it allows for the space between day and time.